tests/activities/models/test_post.py

```diff
@@ -430,6 +430,30 @@ def test_inbound_posts(
     stator.run_single_cycle_sync()
     assert not Post.objects.filter(object_uri="https://remote.test/test-post").exists()
 
+    # Create an inbound new post message with only contentMap
+    message = {
+        "id": "test",
+        "type": "Create",
+        "actor": remote_identity.actor_uri,
+        "object": {
+            "id": "https://remote.test/test-map-only",
+            "type": "Note",
+            "published": "2022-11-13T23:20:16Z",
+            "attributedTo": remote_identity.actor_uri,
+            "contentMap": {"und": "post with only content map"},
+        },
+    }
+    InboxMessage.objects.create(message=message)
+
+    # Run stator and ensure that made the post
+    print("prestat")
```

Comment:
These look like debugging prints left in here!

Reply:
Oh right, yes, that's from copy & pasting the first block in this test. Removed them from both 👍

Reply:
I was going to say, it seems very like something I would do... it was!